server_manager/messages/master_messages.json

```diff
@@ -1099,6 +1109,22 @@
     "message": "Metrics",
     "description": "This string appears within the server view as a header of the section that displays server metrics."
   },
+  "server_metrics_data_transferred": {
```

Comment:
We should decouple the window size from the message. How do we do the current "/ last 30 days"?